src/main/java/org/akip/domain/TaskInstance.java

```diff
@@ -397,33 +401,64 @@ public void setProcessInstance(ProcessInstance processInstance) {
         this.processInstance = processInstance;
     }
 
+    public TaskDefinition getTaskDefinition() {
+        return taskDefinition;
+    }
+
+    public TaskInstance processInstance(TaskDefinition taskDefinition) {
+        this.setTaskDefinition(taskDefinition);
+        return this;
+    }
+
+    public void setTaskDefinition(TaskDefinition taskDefinition) {
+        this.taskDefinition = taskDefinition;
+    }
+
     // jhipster-needle-entity-add-getters-setters - JHipster will add getters and setters here
 
+
     @Override
     public boolean equals(Object o) {
-        if (this == o) {
```

Comment:
Keep the original code. Using only the id field.

Reply:
Done!